Developer

mtsmelo95 opened the pull request.

LucasRCTaborda committed:
.

.

GiselleChaves committed:
teste

GiselleChaves committed:
teste

GiselleChaves committed:
Merge pull request #3 from GiselleChaves/giselle2

testes de push

GiselleChaves committed:
Transforming Department in a Enum Class

GiselleChaves committed:
Menu and department update

GiselleChaves committed:
Task division update

mtsmelo95 committed:
adicionando métodos getTotalCosts, showEmployeeList e totalCostsForTheMonth

mtsmelo95 committed:
adicionando métodos getTotalCosts, showEmployeeList e totalCostsForTheMonth

GustavoDeCampos committed:
Classe para armazenar os custos

GustavoDeCampos committed:
Merge branch 'developer' of https://github.com/GiselleChaves/costManagementSystem into developer

GiselleChaves committed:
Update README.md

GustavoDeCampos committed:
menus

GustavoDeCampos committed:
Merge branch 'developer' of https://github.com/GiselleChaves/costManagementSystem into developer

mtsmelo95 committed:
criando CostRecordList e mudando getTotalCost para CostSystem

mtsmelo95 committed:
criando CostRecordList e mudando getTotalCost para CostSystem

LucasRCTaborda committed:
d

d

Commit:
Metodos a escolha feitos! countEmployeeswithCostOver500 e calculateAverageCostPerEmployee

GiselleChaves committed:
Classes CostSystem, Menu and Cost Record updateds

GiselleChaves reviewed (changes requested):
Commit deve ser feito na branch developer.

GiselleChaves reviewed (changes requested):
Commit deve ser feito na branch developer.

GiselleChaves reviewed (changes requested):
Commit deve ser feito na branch developer.

GiselleChaves reviewed (changes requested):
Commit deve ser feito na branch developer.

GiselleChaves reviewed (changes requested):
Commit deve ser feito na branch developer.

GiselleChaves reviewed (changes requested):
Commit deve ser feito na branch developer.

GiselleChaves reviewed (changes requested):
Commit deve ser feito na branch developer.

GiselleChaves reviewed (changes requested):
Commit deve ser feito na branch developer.

mtsmelo95 committed:
corrigindo método totalCostsForTheMonth

LucasRCTaborda committed:
Uma parte do trabalho foi designada para mim, mas acabei não percebendo que fiz
   o push para a minha branch principal (main) em vez da branch de desenvolvimento (developer).
    Recebi feedback do Mateus de que ele havia revisado e estava tudo bem, no entanto,
     o código não estava na branch de desenvolvimento. Agora percebi que outra pessoa assumiu a funcionalidade.
      Estou preocupado que isso possa me prejudicar, já que era apenas um método simples. Portanto,
      será criada uma terceira funcionalidade para o novo colega de trabalho. Por favor, faça as correções necessárias.

LucasRCTaborda committed:
Merge branch 'developer' of https://github.com/GiselleChaves/costManagementSystem into developer

# Conflicts:
#	src/CostSystem.java

daniellemadrid committed:
Criando método de busca por categoria

GustavoDeCampos committed:
adição do metodo findByDescription

GustavoDeCampos committed:
atualização no menu

GustavoDeCampos committed:
menu

LucasRCTaborda committed:
Alteracao do modelo para main

daniellemadrid committed:
Iniciando o segundo método de verificar funcionários com maiores registros de custos

daniellemadrid committed:
Fazendo um map pra encontrar a maior soma dos registros entre funcionarios

daniellemadrid committed:
Criando um if caso entry for diferente de null e registrar a soma

daniellemadrid committed:
terminando o método de maior registro de custos pelo funcionário

daniellemadrid committed:
Refatorando o método de verificar maior custo registrado por funcionário para deixar mais simples

daniellemadrid committed:
Merge branch 'developer' of https://github.com/GiselleChaves/costManagementSystem into dani

GiselleChaves committed:
Last update os CostSystem Class

GiselleChaves committed:
Last update os CostSystem Class

byGuto committed:
inicio da resolucao do metodo de filtro de gasto por data

byGuto committed:
adicao do for para iterar sobre a lista dos gastos

byGuto committed:
alteracao no nome da variavel da iteracao e mudanca no if que agora se comporta com o ignoreCase

byGuto committed:
adicao do print para mostrar no console a lista de custos naquele mes passado por parametro

byGuto committed:
reset do metodo findCostRecordByDate

byGuto committed:
alterando o nome e o tipo do metodo findCostRecordByDate

byGuto committed:
adicionando uma lista auxiliar e iniciando a iteracao na lista de custos

byGuto committed:
alterando a iteracao do metodo findCostRecordByDate e adicionando o custo na lista auxiliar

GustavoDeCampos committed:
case 7 do menu

GustavoDeCampos committed:
case 8 do menu

GiselleChaves committed:
Last Update of 26/09

GiselleChaves committed:
Merge branch 'developer' of https://github.com/GiselleChaves/costManagementSystem into developer

GustavoDeCampos committed:
lista no construtor

GustavoDeCampos committed:
gustavo

GustavoDeCampos committed:
correções

GustavoDeCampos committed:
mudança metodo

GiselleChaves committed:
Class CostRecord-Case 4 Update

byGuto committed:
construcao do metodo de total de custo dos ultimos 3 meses

byGuto committed:
mudanca no tipo do metodo de total nos ultimos 3 meses e continuacao da construcao

byGuto committed:
iterando sobre a lista e adicionando custos na variavel totalCosts

byGuto committed:
identando e adicionando o return (metodo de total de custo dos ultimos 3 meses)

byGuto committed:
arrumando espacamentos incorretos no metodo

GiselleChaves committed:
Case 4 updated

GiselleChaves committed:
Merge branch 'developer' of https://github.com/GiselleChaves/costManagementSystem into developer

GiselleChaves committed:
Delete method ready

GustavoDeCampos committed:
final menu

GiselleChaves committed:
Last Commit. Menu updated

GiselleChaves committed:
Last Commit. Menu updated
